provider/terraform/utils/provider.go.erb

```diff
@@ -100,10 +101,14 @@ func Provider() terraform.ResourceProvider {
 		},
 
 		ResourcesMap: mergeResourceMaps(
+			<% if !version.nil? && version == 'beta' -%>
```

Comment:
I've been doing `<% unless version.nil? || version == 'ga' -%>` instead since then we don't have to change anything if we introduce an alpha provider.